parent/pom.xml

```diff
@@ -109,17 +122,6 @@
         </repository>
     </distributionManagement>
 
-    <properties>
-        <sonatypeOssDistMgmtSnapshotsUrl>https://jakarta.oss.sonatype.org/content/repositories/snapshots/</sonatypeOssDistMgmtSnapshotsUrl>
-        <sonatypeOssDistMgmtReleasesUrl>https://jakarta.oss.sonatype.org/service/local/staging/deploy/maven2/</sonatypeOssDistMgmtReleasesUrl>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <!--
-          Initialize release.arguments to empty, otherwise if not defined
-          it can fail the release plugin.
-        -->
-        <release.arguments></release.arguments>
-    </properties>
-
```

Comment:
just a nit-pick - why exactly has this code been moved around? It is creating a mess in history (aka `git blame`) as well as the PR harder to review (one needs to go through all lines to find changes). Probably nothing that important within the context of this project but anyway...

Reply:
I'm assuming this nit should not hold up moving forward on this.

Reply:
Think it's better human readable when you declare properties before using them. But this is not important.

Reply:
> Think it's better human readable when you declare properties before using them. 

Agreed!!!